addons/hr/security/hr_security.xml

```diff
@@ -60,5 +60,17 @@
         <field name="model_id" ref="model_hr_plan_activity_type"/>
         <field name="domain_force">['|', ('company_id', '=', False), ('company_id', 'in', company_ids)]</field>
     </record>
+
```

Comment:
We would need to add some rules for responsibles that can be other that employee or manager. For example, when responsible is 'other', we can pick any user; 
In this case, let's say we have responsible 'other', and we pick user without rights on 'hr.employee', we will create a record on 'hr.plan.employee.activity' on which the responsible has no access rights.  :innocent: